src/main/java/bio/terra/service/auth/iam/IamAction.java

```diff
@@ -10,9 +10,13 @@ public enum IamAction {
   // common
   CREATE,
   DELETE,
+  GET,
+  UPDATE,
+  APPROVE,
```

Comment:
What are get and update actions referencing here? Are you adding these actions because in Sam you added them for snapshot settings or for a snapshot or both? Additionally, is approve a common action or should it be listed under a new snapshot request section?

Reply:
So, we don't currently distinguish between what an IamAction refers to. In this case, it's for snapshot_requests. I would say lets start by just adding it to this list and then discuss whether to separate things out.

Reply:
Okay, I know we don't distinguish in the code what the IAmAction refers to, but it seems like there are comment sections here documenting what they refer to. So, I guess I'm still wondering - why don't we want to create a commented section for snapshot requests?

Reply:
Ah! I see. The short comments were disappearing into the IDE hints for me (my brain sorted them into the same category)

Should be good now.